apps/meteor/app/autotranslate/client/lib/actionButton.ts

```diff
@@ -33,18 +34,20 @@ Meteor.startup(() => {
 					const action = 'autoTranslateShowInverse' in message ? '$unset' : '$set';
 					Messages.update({ _id: message._id }, { [action]: { autoTranslateShowInverse: true } });
 				},
-				condition({ message, subscription, user }) {
+				condition({ message, subscription, user, room }) {
 					if (!user) {
 						return false;
 					}
 					const language = subscription?.autoTranslateLanguage || AutoTranslate.getLanguage(message.rid) || '';
+					const isLivechatRoom = roomCoordinator.isLivechatRoom(room?.t);
+					const differentUser = message?.u && message.u._id !== user._id;
```

Comment:
nit picking, but you should probably name this var as `isDifferentUser` for consistency.